### PR TITLE
123

### PR DESCRIPTION
**Pull request by ggevgg**

### CODE REVIEW

**Review by yvhusakov — CHANGES_REQUESTED** [review]
changes

File: README.md [hunk 1]
```diff
@@ -1 +1 @@
-# stat
```
Comment (ggevgg) [inline README.md:1]: bravo!

Comment (ggevgg) [reply README.md:1]: awesome

Comment (ggevgg) [reply README.md:1]: bbb

Comment (mvhusakova) [reply README.md:1]: asd

Comment (mvhusakova) [reply README.md:1]: awesome

**Pull request closed without merging**